src/pages/v2/p/components/V2Dashboard.tsx

```diff
@@ -171,7 +171,14 @@ export default function V2Dashboard({
     ? V2ArchivedProjectIds.includes(projectId) || projectMetadata?.archived
     : false
 
-  const nftsLoading = nftRewardTiersLoading || nftRewardsCIDsLoading
+  // Assumes having `dataSource` means there are NFTs initially
+  // In worst case, if has `dataSource` but isn't for NFTs:
+  //    - loading will be true briefly
+  //    - will resolve false when `useNftRewardTiersOf` fails
```

Comment:
This might seems aids, but I think it's pretty important:
- The longest part of the loading process is getting the CIDs from the `dataSource` contract. 
- So if we wait until the `useNftRewardTiersOf` gives us CIDs, the NFT section will take as long to show up as it is now

Reply:
if its too shitty we can look at caching it somehow in future. lets keep an eye on it.

This seems totally reasonable though.